### PR TITLE
Fix slots, add declarative-shadow-dom if needed

### PR DESCRIPTION
**Pull request by Mihaiii**
For https://github.com/Starcounter/RebelsLounge/issues/152

### CODE REVIEW

File: src/People/wwwroot/People/viewmodels/OrganizationPage.html [hunk 1]
```diff
@@ -66,7 +66,7 @@ <h2 slot="people/organization-heading">Organization details</h2>
         
         <label slot="people/organization-members-label"> Members of this organization: </label>
 
-        <starcounter-include slot="people/organization-members-list" partial="{{model.MembersListPage}}"></starcounter-include>
+        <starcounter-include slot="people/organization-members-control" partial="{{model.MembersListPage}}"></starcounter-include>
```
Comment (Mackiovello) [inline src/People/wwwroot/People/viewmodels/OrganizationPage.html]: Nice catch!

File: src/People/wwwroot/People/viewmodels/OrganizationSmallPage.html [hunk 2]
```diff
@@ -11,4 +11,16 @@
         <input slot="people/organization-firstname-placeholder" type="text" value="{{model.DefaultPerson.FirstName$::change}}" class="form-control" placeholder="First name" />
         <input slot="people/organization-lastname-placeholder" type="text" value="{{model.DefaultPerson.LastName$::change}}" class="form-control" placeholder="Last name" />
```
Comment (Mackiovello) [inline src/People/wwwroot/People/viewmodels/OrganizationSmallPage.html]: I'm not sure why all of these are `type="text"`, but that's not relevant to this PR

**Comment by Mihaiii** [comment]
Conflict fixed.

File: src/People/wwwroot/People/viewmodels/OrganizationSmallPage.html (hunk 2 above)
Comment (warpech) [inline src/People/wwwroot/People/viewmodels/OrganizationSmallPage.html]: What other type could they be?

Comment (Mackiovello) [inline src/People/wwwroot/People/viewmodels/OrganizationSmallPage.html]: House, postal code and phone number should be type `number`,  email address should be type `email`

**Pull request merged**